src/components/Listings/Listings.scss

```diff
@@ -0,0 +1,12 @@
+.listing-section {
```

Comment:
Try to leverage some of the benefits you get from using `SASS` here. Something like:
```css
.listing {
  &-section { ... }
  &-card { ... }
}
```

This helps to consolidate your component-specific styles and scopes them all together.